doc/api/process.md

```diff
@@ -36,6 +36,22 @@ termination, such as calling [`process.exit()`][] or uncaught exceptions.
 The `'beforeExit'` should *not* be used as an alternative to the `'exit'` event
 unless the intention is to schedule additional work.
 
+```js
+process.on('beforeExit', (code) => {
+  console.log('Process beforeExit event with code: ', code);
+});
+
+process.on('exit', (code) => {
+  console.log('Process exit event with code: ', code);
+});
+
+console.log('This message is displayed first.');
+
```

Comment:
Optional suggestion: Add `// Prints:` immediately above line 50.

Reply:
@Trott thanks, I'll take that into consideration for next time but  Rubén suggested remove it: 
[https://github.com/nodejs/node/pull/28430#discussion_r300510283](url)